### ISSUE TITLE
Ability to change location of ~/.npm ~/.cache ~/.config and ~/.local to maven output directory

### ISSUE THREAD

**Issue by dantran** [eirslett/frontend-maven-plugin#163]
Since this plugin is set up to be self contained ( ie not depend on global node and npm), it would be nice to be able to confgure those directories as well.  Those are created by UI build stack

**Comment by eirslett** [comment]
What? Does npm really make all these?

**Comment by dantran** [comment]
~/.npm is the default of npm cache directory.  on CI server, dont think we want to share this
.cache .local and .config are from bower.

user can change these locations using various command line settings and env variable.

Use
XDG_DATA_HOME=xxx to change ~/.local
XDG_CONFIG_HOME=xxx to change ~/.config

I think this plugin allows user to configure ENV vars, user can solve this on their own.  However, it is even better if this can plugin can just abstract out all this settings

**Comment by eirslett** [comment]
Is it possible to configure it in .npmrc and .bowerrc, inside the project directory?

**Comment by dantran** [comment]
yes, I can configure npm and bower cach dir, but not with bower's .local and .config. I am currently using EVN instead

**Comment by dantran** [comment]
I post a questions at bower user group https://groups.google.com/forum/#!topic/twitter-bower/ot3R01Mnj70
